src/components/EntityPage/Participant/ParticipantSummary.js

```diff
@@ -1,32 +1,182 @@
 import * as React from 'react';
 import { compose } from 'recompose';
 import { withTheme } from 'emotion-theming';
-
-import Row from 'uikit/Row';
-import Column from 'uikit/Column';
-
+import VariableSummaryTable from 'uikit/SummaryTable/VariableSummaryTable'
 import { EntityContentSection, EntityContentDivider } from '../';
+import ExternalLink from 'uikit/ExternalLink';
+import { kfWebRoot } from 'common/injectGlobals';
+import { trackUserInteraction, TRACKING_EVENTS } from 'services/analyticsTracking';
+import Holder from "./Utils/Holder";
+import styled from "react-emotion";
+import {Div} from "../../../uikit/Core";
+import SequencingDataTable from "./Utils/SequencingDataTable";
+import OtherDataTypesSummaryTable from "./Utils/OtherDataTypesSummaryTable";
+
+//https://kf-qa.netlify.com/participant/PT_CMB6TASJ#summary
 
 const enhance = compose(withTheme);
 
-const ParticipantSummary = () => {
+/**
+ * Sanitizes an array of SummaryTable data.
+ *
+ * Makes boolean values into their corresponding Strings. Transforms null into "Null value".
+ *
+ * @param arr The array
+ * @returns {*} The sannitized array
+ */
+function sannitize(arr) {
+  return arr.map( ele => {
+    let sannitized = ele.summary;
+    if (ele.summary === null) sannitized = "--";
+    else if(typeof ele.summary === "boolean") sannitized = `${sannitized}`;
+
+    return {title: ele.title, summary: sannitized}
+  })
+}
+
+/**
+ * Sometimes, intermediate nested fields are missing.
+ *
+ * Trying to access participant.diagnoses.hits.edges[0].node.diagnosis_category, for example, crashes the page when
+ * node is missing. https://kf-qa.netlify.com/participant/PT_3FV3E420#summary
+ *
+ * Thus, we're wrapping every access in a try catch that simply returns null when the accessor fails.
+ *
+ * We're taking the opportunity to sannitize the return value:
+ *  null = --
+ *  boolean = true/false
+ *
+ * @param obj
+ * @param accessor
+ * @returns {string|*|null|undefined}
+ */
+function getter(obj, accessor) {
```

Comment:
Only when accessing a property of a property that is an object (including arrays and functions), to avoid `TypeError: can't access property "x" of undefined`

Reply:
Also, It's better to use gt this way:
```
import { get } from 'lodash';

get(...);
```
rather than:
```
import * as _ from 'lodash';
or 
import _ from 'lodash';

_.get(...);
```

This is due to the treeshaking done by Webpack. Webpack won't have to import the whole lodash with the first method: it may have to with the second. It helps to make a smaller package in the end.